packages/gen-ai-hub/src/client/openai/openai-types.ts

```diff
@@ -345,7 +345,7 @@ export interface OpenAiEmbeddingParameters {
 /**
  * Usage statistics for the completion request.
  */
-interface OpenAiUsage {
+export interface OpenAiUsage {
```

Comment:
[q] Is this used somewhere or is it just because it is supposed to be public API?

Reply:
This will be fixed in the other PR (generated client). Ideally all generated types should be exported.